pontoon/base/tests/__init__.py

```diff
@@ -40,6 +40,20 @@ def client_login(self, user=None):
 
         return user
 
+    def assert_redirect_with_error(self, url, message, after_redirect_url='http://testserver/'):
+        """
+        Some of the views are handling errors by returning redirect to the main page
+        and displaying message to the user
+        """
+        response = self.client.get(url)
+
+        assert_code(response, 302)
+        assert_equal(response['Location'], after_redirect_url)
+
+        response = self.client.get('/')
```

Comment:
Asserting things about Django messages is a nightmare, but making a request to the index each time we want to isn't a great solution.

And if we're passing in the response instead, it becomes much harder to mock messages out for testing. Hmm.

I almost think maybe we should just use the built-in `TestCase.assertRedirects` for checking the redirect, and have the test classes mock out the messages stuff and test that instead of trying to make a new assertion method for this.

Or, if we moved this to a more specific subclass for the view tests, it might make sense to pass the URL. I'm not sure, but leaning towards the former. Thoughts?